mwrpy/level2/lev2_collocated.py

```diff
@@ -139,7 +140,41 @@ def generate_lev2_single(
             ):
                 copy_variables(source, nc_output, variables)
 
-            copy_global(nc_lwp, nc_output, nc_lwp.ncattrs())
+            copy_global(nc_lwp, nc_output, tuple(nc_lwp.ncattrs()))
```

Comment:
It's better to relax the type hint instead of casting to tuple:

```python
from collections.abc import Iterable

def copy_variables(source: netCDF4.Dataset, target: netCDF4.Dataset, keys: Iterable[str]) -> None:
    ...

def copy_global(source: netCDF4.Dataset, target: netCDF4.Dataset, attributes: Iterable[str]) -> None:
    ...
```

Reply:
Agreed.